src/main/kotlin/net/maizegenetics/phgv2/api/HaplotypeGraph.kt

```diff
@@ -381,6 +381,21 @@ class HaplotypeGraph(hvcfFiles: List<String>) {
             hapIdToSampleGametes(range).keys.toList()
         }
     }
+    /**
+     * Creates a map of ReferenceRangeId -> (map of hapid -> index)
+     */
```

Comment:
How is the hapIndex determined?  Will that always be a consistent number?

Reply:
Its always consistent because it gets put into a sorted set.  So as long as the hapIdToSampleGametes function does not change which values are returned, it will be consistent.